crates/cadenza-gcode/src/generated/snapshots/cadenza_gcode__generated__test_data__edge_cases__edge_cases_ast.snap

```diff
@@ -0,0 +1,10 @@
+---
+source: crates/cadenza-gcode/src/generated/test_data.rs
+expression: "; Edge cases test\n(Unclosed comment\nN100 G28\nN G28\n(Normal) G1 X10 (Another comment) Y20 ; Semicolon too\n%%\n"
+---
+[
+    [N, 100],
+    [G28],
```

Comment:
Since they're on the same line I would expect the g28 call to be an argument to N. So `[N, 100, [G28]]`. That way the N macro could potentially change the environment before the subcommands execute